racingcar-java/src/test/java/domain/CarTest.java

```diff
@@ -0,0 +1,43 @@
+package domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
+
+class CarTest {
+
+    Car c;
```

Comment:
```suggestion
    private Car c;
```
테스트코드도 항상 접근제어자는 조심해주세요